tilequeue/wof.py

```diff
@@ -892,6 +898,14 @@ def fetch_toi():
                 if n.wof_id in superseded_by_wof_ids:
                     diffs.append((n, None))
```

Comment:
Updated in: https://github.com/mapzen/tilequeue/commit/0757aeaf803530e9c175daddceae0fa3c5733469